code/ecdsap256sha2/Makefile

```diff
@@ -14,7 +14,7 @@ FSTAR_ROOTS += ../tests/Hacl.Test.ECDSA.fst
 
 #BASE_FLAGS+= -funroll-loops 4
 
-CFLAGS += -I ../../../lib/c -march=native -mtune=native -O3
+CFLAGS += -I ../../../lib/c -march=native -mtune=native -O3 -Wno-unused-function
```

Comment:
which function triggers this warning? kremlin should eliminate all unused functions, and if it does not, it's probably a bug -- do you have a repro?